### PR TITLE
Create single orders.md for mobile SDKs

### PR DESCRIPTION
**Pull request by jkim628**
I combined `orders.md` for iOS and Android. I also reworded and moved some things around. I removed `Create Orders` from the documentation. This feature was not supported in Android and was used for testing purposes. If we want partners to create orders via the SDKs in the future, we can implement in Android then add to documentation.

I made some small edits to `README.md`, including replacing the link for `Orders` documentation.

### CODE REVIEW

File: doc/orders.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+```
+
+##### Android
+
+```kotlin
+fun fetchOrders() {
+    FlyBuy.orders.fetch { orders, sdkError ->
+        // Handle orders or deal with error
+    }
+}
```
Comment (csexton) [inline doc/orders.md]: I don't think we need the outer function here, just the method call (to keep it consistent with swift code above)

File: doc/orders.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+```
+
+##### Android
+
+```kotlin
+fun fetchOrder() {
+    FlyBuy.orders.fetch(redemptionCode) { order, sdkError ->
+        // Update order claim view with order details here
+    }
+}
```
Comment (csexton) [inline doc/orders.md]: Same thing here.

File: doc/orders.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+```
+
+## <span id="observe-orders">Observe Orders</span>
```
Comment (csexton) [inline doc/orders.md]: This section is starting to feel very platform specific. Notifications and observables are slightly different between the tow -- do we want to more this to the separate repos?

**Comment by jkim628** [comment]
Agreed that Observe Orders is pretty platform-specific. I can move that to a platform-specific documentation location. Should that go in `flybuy-ios`/`flybuy-android` or should there be iOS and Android folders within `flybuy-documentation`? I think the latter makes more sense to me.

The reason I had iOS and Android separate for methods with the same names, such as `fetch`, is because I wasn't sure if the syntax was specific for Swift vs Kotlin. For example, can Kotlin have `FlyBuy.orders.fetch()`, or does it have to be `FlyBuy.orders.fetch { orders, sdkError -> }`?

**Comment by csexton** [comment]
> The reason I had iOS and Android separate for methods with the same names, such as fetch, is because I wasn't sure if the syntax was specific for Swift vs Kotlin. For example, can Kotlin have FlyBuy.orders.fetch(), or does it have to be FlyBuy.orders.fetch { orders, sdkError -> }?

You are correct, it is specific to the platform -- but you could say call "orders fetch" then provide links to the two SDKs for specifics of the call. 

The reason we want to keep it in the platform code is when we update those method signatures we wanna have one place to update docs. So we don't forget and have wrong/confusing information floating around out there.

**Comment by botnerd** [comment]
The platform-specific documentation is already in the `doc` folder (`sdk/doc` for android) in the `iris-ios` and `iris-android` repos. Those get published to `flybuy-ios` and `flybuy-android`, respectively, when the SDK is published on each.

As far a method code, the syntax may be slightly different per platform, but the names are the same. So, I would think that putting something like `FlyBuy.orders.fetch()` and then saying refer to platform-specific documentation for details would be sufficient.

I think this documentation should focus on how-tos from a higher level. For example, there could be a section on How to Claim an Order that might looks something like this.

**Comment by jkim628** [comment]
Thanks everyone for the input!

After speaking with @botnerd and @jnebeker, I'll update the `orders.md` documents in `flybuy-ios` and `flybuy-android` and keep this repo for more high-level flows.

**Pull request closed without merging**